src/arrayMethodSort.js

```diff
@@ -4,9 +4,37 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
-  };
-}
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    if (String(a) > String(b) || a === undefined) {
+      return 1;
+    }
+
+    if (String(a) === String(b)) {
+      return 0;
+    }
+
+    return -1;
+  }) {
```

Comment:
move it to the separately declared function (with it's own name) instead of writing functionality while declaring parameters